_translator-files/po/de/Client-Troubleshooting.po

```diff
@@ -45,228 +47,230 @@ msgid ""
-msgstr ""
+msgstr "In der Regel ist es viel einfacher und zuverlässiger, ein [Mischpult](https://www.thomann.de/pics/bdb/191244/7355025_800.jpg) zu haben, an das du dein Equipment (Instrumente, Mikrofon, Aufnahmegerät usw.) anschließt und dann ein einfaches Stereosignal an dein Sound-Interface zu senden (achte aber darauf, dass du den resultierenden Sound vom Jamulus Server über deinen Computer abhörst!) Die große Vielfalt an möglichen Hardware-, Software- und Instrumentenkombinationen bedeutet, dass das Einrichten deiner Soundkarte für deine spezielle Konfiguration ansonsten sehr schnell kompliziert werden kann."
 
 #. type: Plain text
 #: ../wiki/en/Client-Troubleshooting.md:68
 msgid "### Somebody joining your jam and being too loud?"
-msgstr ""
+msgstr "### Ist jemand zu laut, wenn er in deine Jam-Session dazukommt?"
 
 #. type: Plain text
 #: ../wiki/en/Client-Troubleshooting.md:70
 msgid "You can set your \"New Client Level\" to a low value (e.g. 10), or set the musicians you are playing with to \"Solo\" state (in the mixer panel on the right side). That way, either new entrants will be very quiet, or you won't hear them at all."
-msgstr ""
+msgstr "Du kannst den \"Pegel für neue Teilnehmer\" auf einen niedrigen Wert (z. B. 10) einstellen oder die Musiker, mit denen du spielst, in den \"SOLO\"-Status versetzen (im Mischpult auf der rechten Seite). Auf diese Weise werden neue Teilnehmer entweder sehr leise sein oder du wirst sie überhaupt nicht hören."
 
 #. type: Plain text
 #: ../wiki/en/Client-Troubleshooting.md:72
 msgid "### Can't see the Server you want to join?"
-msgstr ""
+msgstr "### Du kannst den Server, dem du beitreten möchtest, nicht sehen?"
 
 #. type: Plain text
 #: ../wiki/en/Client-Troubleshooting.md:74
 msgid "First check that you have the right genre Server selected in your Connect window. Sometimes network issues mean your Client won't list all the available Servers. If you know the name of the Server you want to join, you can [look up its IP address here](https://explorer.jamulus.io/). Enter that address in the \"Server Address\" field in the Connect Setup window to connect to it."
-msgstr ""
+msgstr "Vergewissere dich zunächst, dass du den richtigen Genre-Server in deinem Verbindungsfenster ausgewählt hast. Manchmal führen Netzwerkprobleme dazu, dass dein Client nicht alle verfügbaren Server auflistet. Wenn du den Namen des Servers kennst, dem du beitreten möchtest, kannst du [hier seine IP-Adresse nachschlagen](https://explorer.jamulus.io/). Gib diese Adresse in das Feld \"Serveradresse\" im Fenster \"Verbindungsaufbau\" ein, um eine Verbindung herzustellen."
```

Comment:
As noted in other PR, this needs to be updated to Directory not Genre-Server.

Reply:
Corrected